src/App.tsx

```diff
@@ -21,6 +40,10 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
+            value={itemsPerPage}
+            onChange={(event) => {
+              handlerItemsPerPage(event);
+            }}
```

Comment:
` onChange={handlerItemsPerPage}` 
suppose it also will work, and looks shorter

Reply:
fixed